.php-cs-fixer.cache

```diff
@@ -0,0 +1 @@
+{"php":"8.1.5","version":"3.8.0:v3.8.0#cbad1115aac4b5c3c5540e7210d3c9fba2f81fa3","indent":"    ","lineEnding":"\n","rules":{"blank_line_after_opening_tag":true,"braces":{"allow_single_line_anonymous_class_with_empty_body":true},"class_definition":{"inline_constructor_arguments":false,"space_before_parenthesis":true},"compact_nullable_typehint":true,"declare_equal_normalize":true,"lowercase_cast":true,"lowercase_static_reference":true,"new_with_braces":true,"no_blank_lines_after_class_opening":true,"no_leading_import_slash":true,"no_whitespace_in_blank_line":true,"ordered_class_elements":{"order":["use_trait"]},"ordered_imports":{"imports_order":["class","function","const"],"sort_algorithm":"none"},"return_type_declaration":true,"short_scalar_cast":true,"single_blank_line_before_namespace":true,"single_trait_insert_per_statement":true,"ternary_operator_spaces":true,"visibility_required":true,"blank_line_after_namespace":true,"constant_case":true,"elseif":true,"function_declaration":true,"indentation_type":true,"line_ending":true,"lowercase_keywords":true,"method_argument_space":{"on_multiline":"ensure_fully_multiline"},"no_break_comment":true,"no_closing_tag":true,"no_space_around_double_colon":true,"no_spaces_after_function_name":true,"no_spaces_inside_parenthesis":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_class_element_per_statement":{"elements":["property"]},"single_import_per_statement":true,"single_line_after_imports":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"encoding":true,"full_opening_tag":true},"hashes":{"src\/Repository\/PhysicalBookRepository.php":3394572699,"src\/Repository\/BookRepository.php":1712894869,"src\/Repository\/ReviewRepository.php":334335851,"src\/Repository\/BorrowRepository.php":1303677870,"src\/Repository\/UserRepository.php":2069013462,"src\/Entity\/Book.php":1543699175,"src\/Entity\/Review.php":968342023,"src\/Entity\/User.php":3270775170,"src\/Entity\/PhysicalBook.php":1994129303,"src\/Entity\/Borrow.php":37992102,"src\/Controller\/Admin\/ReviewCrudController.php":2828893764,"src\/Controller\/Admin\/PhysicalBookCrudController.php":3779071518,"src\/Controller\/Admin\/BookCrudController.php":4278953303,"src\/Controller\/Admin\/BorrowCrudController.php":2295536312,"src\/Controller\/Admin\/DashboardController.php":443460389,"src\/Controller\/Admin\/UserCrudController.php":572982321,"src\/Kernel.php":4115217777}}
```

Comment:
This file must not be committed and must be added to `.gitignore` (it's weird that Flex didn't add it).

Reply:
This looks like a bug in Flex, because it should add these lines automatically: https://github.com/symfony/recipes/blob/main/friendsofphp/php-cs-fixer/3.0/manifest.json#L6-L9=